clientd3d/dialog.c

```diff
@@ -663,6 +674,41 @@ void DisplayDescription(object_node *obj, BYTE flags, char *description,
 }
 /************************************************************************/
 /*
+* GetRaritySuffix:  Provide a rarity suffix based on the given rarity type.
+*/
+std::string GetRaritySuffix(int rarity) 
+{
+	std::string suffix;
+	
+	switch(rarity)
+	{
+        case ITEM_RARITY_GRADE_NORMAL:
+            suffix = "";
```

Comment:
Just return "" here.

Reply:
Modified 2 instances as requested to `return ""`.  Normal and Default / Unknown rarity.